jmx-scraper/README.md

```diff
@@ -6,3 +6,11 @@ The JMX MBeans and their metric mappings are defined in YAML and reuse implement
 
 This is currently a work-in-progress component not ready to be used in production.
 The end goal is to provide an alternative to the [JMX Gatherer](../jmx-metrics/README.md) utility.
+
+## Component owners
+
+- [Jason Plumb](https://github.com/breedx-splk), Splunk
+- [Robert Niedziela](https://github.com/robsunday), Splunk
```

Comment:
I'll be added to the owners with https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1493 PR, however for Robert you will need to open another PR.
Or alternatively removing him from this readme change until needed.

Reply:
this PR is for `jmx-scraper`, and I think just reflects the existing `component_owners.yml`:

https://github.com/open-telemetry/opentelemetry-java-contrib/blob/d7543676359348d82bdaf644e3e2231ccbcb7e10/.github/component_owners.yml#L44-L47